Template/Pages/index.html

```diff
@@ -1,5 +1,6 @@
 <!--
  Title: Welcome
+ File Name: index
```

Comment:
File names now default to the title unless overridden like this. For all the other files, the title matched. This one needed the override so that it stays as `index.html`, which the server displays by default if no specific file is requested.